dbscripts/xml/upgrade.xml

```diff
@@ -235,6 +235,7 @@
 	</upgrade>
 
 	<upgrade minversion="3.4.0.0" maxversion="3.4.9.9">
+		<migration class="PKP\migration\upgrade\v3_4_0\I9822_ChangeUsageStatsTemporaryTablesIndexes"/>
```

Comment:
I think it's better to add new migrations to the bottom, to avoid breaking expectations of previous migrations (not going to happen here, but...).

Reply:
Oh, yes, I actually always do it -- no clue why I added it here 
Thanks!